akka-docs-dev/rst/scala/http/routing-dsl/directives/security-directives/extractCredentials.rst

```diff
@@ -15,6 +15,8 @@ Description
 Extracts the potentially present ``HttpCredentials`` provided with the request's ``Authorization`` header,
 which can be then used to implement some custom authentication or authorization logic.
 
+See :ref:`credentials-and-timing-attacks-scala` for details about verifying the secret.
```

Comment:
Do we have it mentioned anywhere in the java side of docs? Is there a corresponding API?